extensions/gitpod/src/remoteConnector.ts

```diff
@@ -458,11 +462,13 @@ export default class RemoteConnector extends Disposable {
 			}).connect({
 				host: sshDestInfo.hostName,
 				username: sshDestInfo.user,
-				authHandler(_methodsLeft, _partialSuccess, _callback) {
+				authHandler(methodsLeft, _partialSuccess, _callback) {
+					if (methodsLeft !== null) {
+						return false;
+					}
 					return {
-						type: 'password',
-						username: workspaceId,
-						password: ownerToken,
+						type: 'none',
+						username: sshDestInfo.user,
```

Comment:
Doing more tests seems it's not that easy, using `type: 'none'` and `username` still fails to connect so I guess we need to check for private key files manually :disappointed: and just use `type: 'password'`